experimental/offline_attestation/README.md

```diff
@@ -0,0 +1,110 @@
+# Offline Attestation
+
+The current Oak Functions Runtime implementation uses an interactive handshake
+for setting up an attested end-to-end encrypted channel between the client and
+the runtime. For some scenarios it might be preferable to avoid the handshake.
+
+The offline attestation approach splits the attestation process into two
+distinct parts:
+
+- Key generation and publication: A single asymmetric key-pair is generated. The
```

Comment:
Sometimes "exporting" is used instead of "publication" to describe this concept, up to you if you think that helps.